### PR TITLE
docs: improved `plugin-develop` doc

### LINKED ISSUE #5870: docs: add enable commands to the control API in the plugin development documentation
### Improved or typo fixed.

Hi, community, 
When I develop a plugin based on the [plugin development documentation](https://apisix.apache.org/docs/apisix/plugin-develop/#register-control-api), in the case of the control API, there is no relevant command in the documentation. I am still using port 9080 to test the control API, but I keep getting 404 errors.

Then I found out in the [control API documentation](https://apisix.apache.org/docs/apisix/control-api) that it was listening on port 9090 and I knew I should test it with 9090. But I had never thought of it as a port problem before.

### Solution

We can write a test command in the plugin development documentation under the control API, like
```
curl -i http://127.0.0.1:9090/v1/plugin/example-plugin/hello
```
and add a tip to remind the user that the control API is listening on port 9090
image

### PR DESCRIPTION
**Pull request by guoqqqi**
### What this PR does / why we need it:
resolve #5870 

Added control API test commands to the plugin developer documentation

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/en/latest/plugin-develop.md [hunk 1]
```diff
@@ -493,4 +493,12 @@ function _M.control_api()
 end
 ```
 
-If you don't change the default control API configuration, the plugin will be expose `GET /v1/plugin/example-plugin/hello` which can only be accessed via `127.0.0.1`.
+If you don't change the default control API configuration, the plugin will be expose `GET /v1/plugin/example-plugin/hello` which can only be accessed via `127.0.0.1`. Test with the following command:
```
Comment (tokers) [inline docs/en/latest/plugin-develop.md:496]:
```suggestion
If you don't change the default control API configuration, the plugin will expose `GET /v1/plugin/example-plugin/hello` which can only be accessed via `127.0.0.1`. Test with the following command:
```

**Review by tokers — APPROVED** [review]

File: docs/en/latest/plugin-develop.md [hunk 2]
```diff
@@ -493,4 +493,12 @@ function _M.control_api()
...
 
-If you don't change the default control API configuration, the plugin will be expose `GET /v1/plugin/example-plugin/hello` which can only be accessed via `127.0.0.1`.
+If you don't change the default control API configuration, the plugin will expose `GET /v1/plugin/example-plugin/hello` which can only be accessed via `127.0.0.1`. Test with the following command:
+
+```shell
+curl -i -X GET "http://localhost:9090/v1/plugin/example-plugin/hello"
+```
+
+:::note Note
+By default, the control API server is enabled and listens to `127.0.0.1:9090`. You can change it via the `control` section under `apisix` in `conf/config.yaml`: [Read more about control API introduction](./control-api)
```
Comment (spacewander) [inline docs/en/latest/plugin-develop.md:503]: Missing a `.md` in the link? And better to give an example of the new configuration.

File: docs/en/latest/plugin-develop.md [hunk 3]
```diff
@@ -493,4 +493,12 @@ function _M.control_api()
...
 ```
 
-If you don't change the default control API configuration, the plugin will be expose `GET /v1/plugin/example-plugin/hello` which can only be accessed via `127.0.0.1`.
+If you don't change the default control API configuration, the plugin will expose `GET /v1/plugin/example-plugin/hello` which can only be accessed via `127.0.0.1`. Test with the following command:
+
+```shell
+curl -i -X GET "http://localhost:9090/v1/plugin/example-plugin/hello"
+```
+
+:::note Note
```
Comment (spacewander) [inline docs/en/latest/plugin-develop.md:502]: Is it necessary to use `:::note`? If not, we can remove it.

File: docs/en/latest/plugin-develop.md [hunk 4]
```diff
@@ -493,4 +493,12 @@ function _M.control_api()
 end
 ```
 
-If you don't change the default control API configuration, the plugin will be expose `GET /v1/plugin/example-plugin/hello` which can only be accessed via `127.0.0.1`.
+If you don't change the default control API configuration, the plugin will expose `GET /v1/plugin/example-plugin/hello` which can only be accessed via `127.0.0.1`. Test with the following command:
+
+```shell
+curl -i -X GET "http://localhost:9090/v1/plugin/example-plugin/hello"
```
Comment (spacewander) [inline docs/en/latest/plugin-develop.md:499]: There is no requirement that `localhost` should be always mapped to `127.0.0.1`.

File: docs/en/latest/plugin-develop.md (hunk 3 above)
Comment (spacewander) [reply docs/en/latest/plugin-develop.md:502]: To me, this is just a regular tip. Better leave it for something really critical.

**Review by spacewander — APPROVED** [review]

**Pull request merged**